src/main/java/org/ukma/spring/crooodle/dto/RoomCrudRequestDto.java

```diff
@@ -0,0 +1,4 @@
+package org.ukma.spring.crooodle.dto;
+
+public class RoomCrudRequestDto {
```

Comment:
There is no really such as a thing as CRUD request.
Delete simply takes an id.
Create or update might or might not be represented as a single DTO.
Having separate update DTO would allow for having partial editing,
where all fields except the edited one is null.